tests/test_legacy_api.py

```diff
@@ -1183,6 +1183,51 @@
         "events",
         id="dots_in_alias",
     ),
+    pytest.param(
+        {
+            "project": [1],
+            "dataset": "events",
+            "from_date": "2021-04-01T20:05:27",
+            "to_date": "2021-04-15T20:05:27",
+            "groupby": ["group_id"],
+            "conditions": [
+                [["positionCaseInsensitive", ["message", "'Api\\'"]], "!=", 0],
```

Comment:
I think this is a Sentry bug by not escaping the trailing backslash, but it's easier to fix it here then try to dig into the internals of issues and group serialization. It's a VERY rare edge case.